driver/docker-container/driver.go

```diff
@@ -387,13 +387,19 @@ func (d *Driver) Factory() driver.Factory {
 	return d.factory
 }
 
-func (d *Driver) Features() map[driver.Feature]bool {
+func (d *Driver) Features(ctx context.Context) map[driver.Feature]bool {
+	var historyAPI bool
+	c, err := d.Client(ctx)
```

Comment:
This call is expensive. It makes an new `docker exec`.

Reply:
The client is cached https://github.com/docker/buildx/blob/a906149930ebe4dbd33d96bd42e157a661956d4c/driver/manager.go#L157-L162 so we are probably fine here no?

Reply:
That is not the struct that's get called in the line 392.